AppCenter/AppCenter.xcodeproj/project.pbxproj

```diff
@@ -230,6 +230,12 @@
 		6E48A5A41D3831FE006E8B5F /* MSChannelUnitConfigurationTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 6E48A5A31D3831FE006E8B5F /* MSChannelUnitConfigurationTests.m */; };
 		6E48A5A71D383893006E8B5F /* MSChannelGroupDefaultTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 6E48A5A61D383893006E8B5F /* MSChannelGroupDefaultTests.m */; };
 		6EB1F40E1D2443B7005F9F99 /* MSChannelUnitDefaultTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 6EB1F40D1D2443B7005F9F99 /* MSChannelUnitDefaultTests.m */; };
+		7839727423B39C2A00620142 /* MSAlertController.h in Headers */ = {isa = PBXBuildFile; fileRef = 78686FAB23B390C500CDA4C1 /* MSAlertController.h */; };
+		7839727623B39C2B00620142 /* MSAlertController.h in Headers */ = {isa = PBXBuildFile; fileRef = 78686FAB23B390C500CDA4C1 /* MSAlertController.h */; };
```

Comment:
On another topic the alert logic has always been in a "vendor" folder. I am not sure why we did this in the past, I don't strongly about this but I want to make sure we are not missing something by moving it. Basically just raising awareness.

Reply:
(moved because we need some SDK utils as dependency there)